cfme/control/explorer/conditions.py

```diff
@@ -77,8 +76,8 @@ class ConditionFormCommon(ControlExplorerView):
 
     title = Text("#explorer_title_text")
     description = Input(name="description")
-    scope = ExpressionEditor("//img[@alt='Edit this Scope']")
-    expression = ExpressionEditor("//img[@alt='Edit this Expression']")
+    scope = ExpressionEditor(".//button[normalize-space(.)='Define Scope']")
```

Comment:
If you don't use relative xpath here, you shouldn't have to modify `ExpressionEditor.enable_editor`